React/Modules/RCTHapticFeedback.m

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTHapticFeedback.h"
+#import <UIKit/UIKit.h>
+
+@implementation RCTHapticFeedback
+
+{
+  UIImpactFeedbackGenerator *_impactFeedback;
+  UINotificationFeedbackGenerator *_notificationFeedback;
+  UISelectionFeedbackGenerator *_selectionFeedback;
+}
+@synthesize bridge = _bridge;
+
+RCT_EXPORT_MODULE()
+
+- (instancetype)init
+{
+  if ((self = [super init])) {
+    
+  }
+  return self;
```

Comment:
This should actually be removed, we don't want to have an init implemented to make the module lazy initiable.